Project.toml

```
@@ -20,6 +20,6 @@ OrdinaryDiffEq = "6"
 PGFPlotsX = "1"
 Plots = "1"
 PyPlot = "2"
-Trixi = "0.4, 0.5"
+Trixi = "0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10, 0.11"
```

Are you sure about this? I'd probably restrict it to more or less current versions (and eventually you should add a downgrade check)
```suggestion
Trixi = "0.9 - 0.11"
```

---

I will double check this. Adding a Downgrade to the CI as well as additional CI for other OS types will be added to an issue.

---

Bumping to more recent versions of Trixi means I would need to bump up the required Julia version as well.

---

I am going to leave this be for the time being. When I attempted to change it I broke everything as the CI said there were unresolvable dependencies. Also, the Trixi2Vtk deps list includes older Trixi versions so it might be the work idea (at least for the near future)